### ISSUE TITLE
[BUG][WEBJS] - Interface/api/sendText reply_to message, click the message abnormally in the official app or web terminal

### ISSUE THREAD

**Issue by TimousFly** [allburov/whatsapp-http-api#542]
In a direct chat, I use the /api/sendText interface and the replay_to field. The other party replies to a message I sent. When I click on the reply message on my phone, I cannot locate the message I sent.

**Comment by devlikepro** [comment]
yeah, it's minor bug, kinda weird one, likely the payload is not fully compatible with the payload WA expects

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by LouisAldorio** [comment]
I tried to run the image with command


docker run devlikeapro/waha:latest


I get error:

exec /usr/local/bin/docker-entrypoint.sh: exec format error

seems like the latest docker image got problem.

**Comment by devlikepro** [comment]
@LouisAldorio hi! 
Could you give some info about this in separate issue?
1. Your OS + version
2. `docker info` output

Isn't it macos by any chance?
https://github.com/docker/for-mac/issues/7168
https://github.com/docker/for-mac/issues/7168#issuecomment-1929204756

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
There's also arm version (and you can use any previous version meanwhile)
https://waha.devlike.pro/docs/how-to/engines/#docker-images
```
devlikeapro/waha:arm
```

or
```
devlikeapro/waha:latest-2024.9.4
```

@LouisAldorio try those and we'd love to hear back about any issues (or successes)!


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by LouisAldorio** [comment]
It is in WSL2 ubuntu inside windows 11

Here is the docker info

Client: Docker Engine - Community
 Version:    24.0.7
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.11.2
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.21.0
    Path:     /usr/libexec/docker/cli-plugins/docker-compose

Server:
 Containers: 5
  Running: 0
  Paused: 0
  Stopped: 5
 Images: 36
 Server Version: 24.0.7
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc io.containerd.runc.v2
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 3dd1e886e55dd695541fdcd67420c2888645a495
 runc version: v1.1.10-0-g18a0cb0
 init version: de40ad0
 Security Options:
  seccomp
   Profile: builtin
 Kernel Version: 5.10.16.3-microsoft-standard-WSL2
 Operating System: Ubuntu 22.04.3 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 12
 Total Memory: 7.586GiB
 Name: LAPTOP-CSK4B7V1
 ID: d1b7063b-1304-40b1-96e2-4ee49582518d
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Username: louisaldorio
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

**Comment by LouisAldorio** [comment]
devlikeapro/waha:latest-2024.9.4

this image works like charm anyway

**Comment by devlikepro** [comment]
@LouisAldorio try switching to **hyper-v backend** so it uses normal linux core , not ms built-in one
https://docs.docker.com/desktop/install/windows-install/


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by LouisAldorio** [comment]
I tried to build yout code to a docker image
 
 but I get the same error again using the Dockerfile in core branch
 
which Dockerfile you use to build tag latest-2024.9.4 ?
